pybamm/models/submodels/interface/open_circuit_potential/plett_ocp.py

```diff
@@ -0,0 +1,114 @@
+#
+# OCP with empiral hysteresis function, and hysteresis state function
+#
+import pybamm
+from . import BaseOpenCircuitPotential
+
+
+class PlettOpenCircuitPotential(BaseOpenCircuitPotential):
```

Comment:
can you document the Plett paper this is taken from?

Reply:
string in `.register()` needs to match the name of the entry in the .bib file

Reply:
The citation registration was placed in section which checks the model options